packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/CopyView.tsx

```diff
@@ -98,13 +94,13 @@ CopyView.Provider = CopyViewProvider;
 CopyView.Cancel = ActionCancelControl;
 CopyView.Destination = ActionDestinationControl;
 CopyView.Exit = ActionExitControl;
-CopyView.Folders = FoldersTableControl;
-CopyView.FoldersLoading = LoadingIndicatorControl;
 CopyView.FoldersMessage = FoldersMessageControl;
 CopyView.FoldersPagination = FoldersPaginationControl;
 CopyView.FoldersSearch = SearchControl;
+CopyView.FoldersTable = FoldersTableControl;
+CopyView.LoadingIndicator = LoadingIndicatorControl;
```

Comment:
Think this needs to include "Folder" in the name still since it is only relevant to the folder loading